.travis.yml

```diff
@@ -1,5 +1,16 @@
-before_install: bash ./scripts/validate-tests.sh
-after_success: bash ./scripts/docs-upload-gh.sh
+before_install:
+  # Repo for Yarn
+  - sudo apt-key adv --fetch-keys http://dl.yarnpkg.com/debian/pubkey.gpg
```

Comment:
`npm i yarn` should actually work -- this is just the recommended path to getting it installed per the Yarn docs: https://yarnpkg.com/en/docs/install-ci#travis-tab

I think it's probably just a matter of the speed of getting the binary onto the box.